packages/web-components/README.md

```diff
@@ -3,7 +3,7 @@
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 [![npm version](https://badge.fury.io/js/%40microsoft%2Fweb-components.svg)](https://badge.fury.io/js/%40microsoft%2Fweb-components)
 
-`@fluentui/web-components` is a library of Web Components that _composes_ `@microsoft/fast-foundation` and supports Microsoft's Fluent design language.
+`@fluentui/web-components` is a library of web components that _composes_ `@microsoft/fast-foundation` and supports Microsoft's Fluent design language.
```

Comment:
Assuming this is a test? :)

I think we want to keep this formatting as FYI. Let me know if you need help digging in here as well!

Reply:
no problem; this is just a test, i won't merge. Trying to validate build with lage (which enables selective build so we don't need to build the whole repo on changes)